seed/static/seed/js/controllers/analyses_controller.js

```diff
@@ -38,6 +38,9 @@ angular.module('BE.seed.controller.analyses', [])
       }
 
       $scope.start_analysis = function(analysis_id) {
+        analysis = $scope.analyses.find(function(a) { return a.id === analysis_id })
+        analysis.status = 'Starting...'
```

Comment:
added this to have the side effect of removing the "start" button once the click it